.gitignore

```diff
@@ -63,3 +63,15 @@ coverage.xml
 
 # Sphinx documentation
 doc/_build/
+.vscode/
+<<<<<<< HEAD
+<<<<<<< HEAD
+Pipfile Pipfile.lock
+=======
+Pipfile
+Pipfile.lock
+>>>>>>> Formatting with yapf applied.\nVersion reverted to orig version 0.5.0+dev\nPipenv files added to gitignore
+=======
+Pipfile
+Pipfile.lock
+>>>>>>> 47197bf4922eb92510defd465e78bb6d375ccbe9
```

Comment:
Yes, I had something like this with the version file as well. As I said the rebase was frantic. I might have typed the wrong command. 
What should I do to prevent this the next time and more important fix this. What worries me is that this went undetected by the testing. Is there some kind of test configuration testing?

Reply:
We don't have any testing for `.gitignore`. I'm not sure how to do that.

I guess we could have a test for conflict markers like this being checked in. In practice it's unlikely to slip by reviewers, though, so it doesn't seem like a high priority.